intermediate-python-debugging/debugging-with-vs-code.md

```diff
@@ -95,16 +95,16 @@ Notice that the program will pause at the first breakpoint it encounters.
 
 You can create a watch by clicking on the `+` icon and enter any expression.
 
-![Entering a Watch](../assets/vs-code-debugger/watch-price.png)
+![Entering a Watch](../assets/vs-code-debugger/watch-guess-before.png)  
 *Fig. Creating a watch*
 
-![Watches](../assets/vs-code-debugger/watch.png)
+![Watches](../assets/vs-code-debugger/watch-guess-after.png)  
```

Comment:
I also like that you updated these with examples that make more sense given the sample code.